tag.sh

```diff
@@ -19,11 +19,9 @@ for dir in ${VERSIONS}; do
   date_and_hash="${commit_date}-$(git rev-parse --short HEAD)"
 
   full_reg_name="$REGISTRY$name"
-  echo "-> Tagging image '$IMAGE_ID' as '$full_reg_name:$version' and '$full_reg_name:latest' and '$full_reg_name:$date_and_hash'"
-  if [[ $OS == "c9s" ]] || [[ $OS == "fedora" ]]; then
-    echo "-> Tagging image '$IMAGE_ID' as '$full_reg_name:$OS'"
-    docker tag "$IMAGE_ID" "$full_reg_name:$OS"
-  fi
+  echo "-> Tagging image '$IMAGE_ID' as '$full_reg_name:$version' and '$full_reg_name:latest' and '$full_reg_name:$OS' and '$full_reg_name:$date_and_hash'"
```

Comment:
Could you please explain this change?
I would like to double-check because it affects also other distributions.

Reply:
It does not make sense to tag image as `:$OS` only for c9s, c8s, and fedora. Let's do it for all images. But only c8s, c9s and fedora use it.